src/main/groovy/com/ullink/gradle/nunit/ReportGenerator.groovy

```diff
@@ -0,0 +1,116 @@
+package com.ullink.gradle.nunit
+
+import org.gradle.api.tasks.Exec
+
+class ReportGenerator extends Exec {
+
+    def ExtentReportType = 'v3html'
+    def ExtentReportDownloadUrl = 'https://github.com/extent-framework/extentreports-dotnet-cli/archive'
+    def ExtentReportZipName = 'master.zip'
+    def ExtentReportExecutablePath = 'extentreports-dotnet-cli-master/dist/extent.exe'
+
+    @Override
+    protected void exec() {
+        if (isTestResultFileAvailable()) {
+            project.logger.info("Generating the report for the Test Results..")
```

Comment:
[minor] Maybe log the `nunit` word too ("nunit test results")?

Reply:
Done